modules/ROOT/partials/configuration/rtc_user_details_provider.adoc

```diff
@@ -5,7 +5,7 @@ By default, a user's unique ID (`userId`, the `sub` field from their xref:rtc/jw
 
 To display a descriptive name on the caret, the `userId` needs to be resolved into user details that include the user's display name (`fullName`). This resolution is done on each client to avoid sending any personal information through the RTC server.
 
-Only the `fullName` is required, but the whole object will be stored within the RTC caret information for use with the <<rtc_client_connected,`rtc_client_connected`>> and <<rtc_client_disconnected,`rtc_client_disconnected`>> callbacks to avoid duplicate lookup queries. The user details are also passed to the `RtcClientConnected` and `RtcClientDisconnected` events.
+Only the `fullName` is required, but the whole object will be stored within the RTC caret information for use with the xref:rtc_client_connected[`rtc_client_connected`]and xref:rtc_client_disconnected[]`rtc_client_disconnected`] callbacks to avoid duplicate lookup queries. The user details are also passed to the `RtcClientConnected` and `RtcClientDisconnected` events.
```

Comment:
```suggestion
Only the `fullName` is required, but the whole object will be stored within the RTC caret information for use with the xref:rtc_client_connected[`rtc_client_connected`] and xref:rtc_client_disconnected[`rtc_client_disconnected`] callbacks to avoid duplicate lookup queries. The user details are also passed to the `RtcClientConnected` and `RtcClientDisconnected` events.
```